src/Markdig/Parsers/FencedBlockParserBase.cs

```diff
@@ -191,14 +179,7 @@ public override BlockState TryOpen(BlockProcessor processor)
             // Add the language as an attribute by default
             if (!string.IsNullOrEmpty(fenced.Info))
             {
-                if (string.IsNullOrEmpty(InfoPrefix))
```

Comment:
Don't recall and can't see it from the PR context, but InfoPrefix can never be empty?

Reply:
I think what he was getting at is that string concat with null or empty is basically a nop, so the if/else can be simplified.

Reply:
Indeed. string.Concat(string, string) tests this already.